server/consumer.go

```diff
@@ -1644,6 +1644,22 @@ func (o *Consumer) SetInActiveDeleteThreshold(dthresh time.Duration) error {
 	return nil
 }
 
+// switchToEphemeral is called on startup when recovering ephermals.
```

Comment:
seems good other than typo here :)

LGTM

Reply:
thx will fix and merge.